.gitignore

```diff
@@ -50,3 +50,6 @@ junit.xml
 /k8sfv/prometheus/data
 /k8sfv/prometheus/prometheus.yml
 /k8sfv/output/
+
+# tools
+manifest-tool
```

Comment:
We don't need this anymore as we are using this tool inside the container.

Reply:
Good point. Updating and squashing.